homeassistant/components/media_player/sonos.py

```diff
@@ -909,6 +910,8 @@ def play_media(self, media_type, media_id, **kwargs):
                     _LOGGER.error('Error parsing media uri "%s", '
                                   "please check it's a valid media resource "
                                   'supported by Sonos', media_id)
+            elif media_type == MEDIA_TYPE_SPEECH:
```

Comment:
Should we have a specific Speech type?
Would it not be better to use a more generic audio type?